packages/graphql/src/entities/GitInfo.ts

```diff
@@ -0,0 +1,35 @@
+import { nxs, NxsResult } from 'nexus-decorators'
+import type { BaseContext } from '../context'
+import type { GitInfo as GetGitInfo } from '@packages/server/lib/util/git'
```

Comment:
I think we said that we're not importing from `@packages/server` in `@packages/graphql`

Reply:
Just the type - I could move this to `@packages/types`.

Reply:
I think IIRC the issue is then when we import from `@packages/server` we get globals that otherwise cause interference & require us to do `skipLibCheck`

Reply:
ok, done